oml/datasets/base.py

```diff
@@ -112,7 +112,9 @@ def __init__(
         self.df = df
         self.transform = transform if transform else get_transforms("norm_albu")
         self.f_imread = f_imread
-        self.read_bytes_image_cached = lru_cache(maxsize=cache_size)(self._read_bytes_image)
+        self.read_bytes_image_cached = (
```

Comment:
if you don't want to use cache you still have a function that has "cached" in the name 
let's rename it?